test/run-drun/max-stack-variant.mo

```diff
@@ -2,7 +2,7 @@
 import { errorMessage; debugPrint; } = "mo:⛔";
 
 actor {
-    let expectedMinimumSize = 31_000;
+    let expectedMinimumSize = 29_000;
```

Comment:
Not clear why this has changed:
Maybe the Wasm stack occupation changed with the recent IC and more recent `wasmtime` version?